### ISSUE TITLE
Command Line Usage?

### ISSUE THREAD

**Issue by JamoCA** [gcardone/junidecode#11]
What would be required in order to use Junidecode at the command line?

I use [CFLint](https://github.com/cflint/CFLint) with [VSCode](https://code.visualstudio.com/) to identify issues with my CFML code.  While using pasting content provided by clients into VSCode, I'd like to replace special characters like curly quotes, emdashes, etc.  I'm currently using JUnidecode with Tomcat to sanitize content provided via HTTP, but would like to extend that to VSCode.  (NOTE:  I've checked and there's nothing that can intelligently convert text to ASCII7 in the VSCode Marketplace.)

Is it possible to update JUnidecode so that a file (or text string) can be converted to ASCII7 at the command line like this:

    java junidecode-0.4.1.jar -file <fullPathToFile>

or 

    java junidecode-0.4.1.jar <textStringToConvert>

Thanks!

**Comment by gcardone** [comment]
The junidecode jar already supports reading from stdin or from a string argument:

```
$ gradle jar

BUILD SUCCESSFUL in 0s
2 actionable tasks: 2 executed
$ java -jar build/libs/junidecode-0.5.jar "hèlló ẁörld"
hello world
$ echo "hèlló ẁörld" | java -jar build/libs/junidecode-0.5.jar
hello world
```

I hope this helps.

**Comment by JamoCA** [comment]
Hmm....  I used the command line w/Windows 10 with the publicly available 4.1 and received "hello ?orld".  (The "w" wasn't successfully converted.)

Do "quotes" have to be used?  (I noticed that they weren't returned.)  This would mean that special care would need to be done in order to preserve existing quotes.

What if I want to do a block of text?  Or the text contains carraige returns which aren't necessarily command-line-friendly?  is there a way to loop over a text file and have it converted line-by-line?
